crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -69,15 +68,14 @@ pub mod call {
 }
 
 /// Used to deserialize a reply from [Client::code](crate::sequencer::Client::code).
-#[serde_as]
-#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(deny_unknown_fields)]
-pub struct Code {
+pub struct Code<'a> {
     // Unknown block hash results in empty abi represented as a JSON
     // object, instead of a JSON array
-    #[serde_as(deserialize_as = "DefaultOnError")]
-    pub abi: Vec<code::Abi>,
-    pub bytecode: Vec<ByteCodeWord>,
+    #[serde(borrow)]
+    pub abi: &'a serde_json::value::RawValue,
+    pub bytecode: Vec<StarkHash>,
```

Comment:
I'm unsure what the `#[serde_as(deserialize_as = "DefaultOnError")]` was achieving. And I'm not sure how to get it back.

Reply:
There's no need if you're using a raw value :+1: